.github/workflows/integrationTest.yml

```diff
@@ -81,47 +81,47 @@ jobs:
           { os: "rhel8-2", package: "package-rpm",  username: "ec2-user",
             installAgentCommand: "rpm -U ./amazon-cloudwatch-agent.rpm",
-            ami: "cloudwatch-agent-integration-test-rhel8-2", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
+            ami: "cloudwatch-agent-integration-test-rhel8-2*", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
             arc: "amd64", binaryName: "amazon-cloudwatch-agent.rpm" },
           { os: "rhel8-3", package: "package-rpm",  username: "ec2-user",
             installAgentCommand: "rpm -U ./amazon-cloudwatch-agent.rpm",
-            ami: "cloudwatch-agent-integration-test-rhel8-3", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
+            ami: "cloudwatch-agent-integration-test-rhel8-3*", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
             arc: "amd64", binaryName: "amazon-cloudwatch-agent.rpm" },
           { os: "rhel8-4", package: "package-rpm",  username: "ec2-user",
             installAgentCommand: "rpm -U ./amazon-cloudwatch-agent.rpm",
-            ami: "cloudwatch-agent-integration-test-rhel8-4", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
+            ami: "cloudwatch-agent-integration-test-rhel8-4*", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
             arc: "amd64", binaryName: "amazon-cloudwatch-agent.rpm" },
           { os: "ol8-1", package: "package-rpm",  username: "ec2-user",
             installAgentCommand: "rpm -U ./amazon-cloudwatch-agent.rpm",
-            ami: "cloudwatch-agent-integration-test-ol8-1", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
+            ami: "cloudwatch-agent-integration-test-ol8-1*", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
             arc: "amd64", binaryName: "amazon-cloudwatch-agent.rpm" },
           { os: "ol8-2", package: "package-rpm",  username: "ec2-user",
             installAgentCommand: "rpm -U ./amazon-cloudwatch-agent.rpm",
-            ami: "cloudwatch-agent-integration-test-ol8-2", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
+            ami: "cloudwatch-agent-integration-test-ol8-2*", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
             arc: "amd64", binaryName: "amazon-cloudwatch-agent.rpm" },
           { os: "o8-3", package: "package-rpm",  username: "ec2-user",
             installAgentCommand: "rpm -U ./amazon-cloudwatch-agent.rpm",
-            ami: "cloudwatch-agent-integration-test-ol8-3", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
+            ami: "cloudwatch-agent-integration-test-ol8-3*", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
             arc: "amd64", binaryName: "amazon-cloudwatch-agent.rpm" },
           { os: "ol8-4", package: "package-rpm",  username: "ec2-user",
             installAgentCommand: "rpm -U ./amazon-cloudwatch-agent.rpm",
-            ami: "cloudwatch-agent-integration-test-ol8-4", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
+            ami: "cloudwatch-agent-integration-test-ol8-4*", caCertPath: "/etc/ssl/certs/ca-bundle.crt",
```

Comment:
Trying to understand how the * wildcard is used here. Does this pick up any random AMI that starts with this substring?

Reply:
In the main it picks latest that matches the name

Reply:
https://github.com/aws/amazon-cloudwatch-agent/blob/master/integration/terraform/ec2/linux/main.tf#L42